### ISSUE TITLE
Insufficient contrast ratio on hover

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3111]
The menu items have white text (#FFF) displayed on a black background (#000) with the keyboard shortcuts displayed in gray (#999). On hover, the item's background color changes to blue (#3DBDEE) which has a contrast ratio of 2.16:1 against the white text  and 1.31:1 against the gray keyboard shortcuts, below the 4.5:1 minimum contrast  requirement for normal-sized text. 

**Suggested Solution** 
Either use a different hover indicator or a different background color that has a contrast ratio of at least 4.5:1 against the white and gray text. Note that darkening the blue color to achieve a sufficient contrast ratio will result in an undistinguishable hover indicator with low contrast against the surrounding black items. 

Severity: 2
